tests/unittests/benchmark/test_assessments.py

```diff
@@ -0,0 +1,192 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""Tests for :mod:`orion.benchmark.assessment`."""
+
+import plotly
+
+import orion.core.io.experiment_builder as experiment_builder
+from orion.benchmark.assessment import AverageRank, AverageResult
+from orion.testing import OrionState, create_experiment, generate_trials
+
+config = dict(
+    name="experiment-name",
+    space={"x": "uniform(0, 200)"},
+    metadata={
+        "user": "test-user",
+        "orion_version": "XYZ",
+        "VCS": {
+            "type": "git",
+            "is_dirty": False,
+            "HEAD_sha": "test",
+            "active_branch": None,
+            "diff_sha": "diff",
+        },
+    },
+    version=1,
+    pool_size=1,
+    max_trials=10,
+    working_dir="",
+    algorithms={"random": {"seed": 1}},
+    producer={"strategy": "NoParallelStrategy"},
+)
+
+trial_config = {
+    "experiment": 0,
+    "status": "completed",
+    "worker": None,
+    "start_time": None,
+    "end_time": None,
+    "heartbeat": None,
+    "results": [],
+    "params": [],
+}
+
+algorithms = [{"random": {"seed": 1}}, {"tpe": {"seed": 1}}]
+
+
+class TestAverageRank:
+    """Test assessment AverageRank"""
+
+    def test_creation(self):
+        """Test creation"""
+        ar1 = AverageRank()
+        assert ar1.task_num == 1
+        assert ar1.configuration == {
+            "orion-benchmark-assessment-averagerank-AverageRank": {"task_num": 1}
+        }
+
+        ar2 = AverageRank(task_num=5)
+        assert ar2.task_num == 5
+        assert ar2.configuration == {
+            "orion-benchmark-assessment-averagerank-AverageRank": {"task_num": 5}
+        }
+
+    def test_plot_figures(self):
+        """Test assessment plot"""
+        ar1 = AverageRank()
+
+        with create_experiment(config, trial_config, ["completed"]) as (
+            _,
+            experiment,
+            _,
+        ):
+            plot = ar1.plot_figures("task_name", [(0, experiment)])
+
+        assert type(plot) is plotly.graph_objects.Figure
+
+    def test_figure_layout(self):
+        """Test assessment plot format"""
+        ar1 = AverageRank()
+
+        gen_exps = []
```

Comment:
Could we move the code here for the creation of the experiments outside of this tests so that it can be reused for other assessment test of figure layout.